src/lib/Components/Bidding/Screens/__tests__/BidResult-tests.tsx

```diff
@@ -4,22 +4,46 @@ import "react-native"
 import * as renderer from "react-test-renderer"
 import { BidResult } from "../BidResult"
 
-it("renders winning screen properly", () => {
-  jest.useFakeTimers()
+const saleArtwork = {
+  current_bid: {
+    amount: "CHF10,000",
+    cents: 1000000,
+    display: "CHF 10,000",
+  },
+  sale: {
+    live_start_at: "2022-01-01T00:03:00+00:00",
+    end_at: "2022-05-01T00:03:00+00:00",
+  },
+}
+describe("BidResult component", () => {
+  Date.now = jest.fn(() => 1525983752116)
```

Comment:
Let's put this in an `beforeEach` block. I mean, I guess it's a matter of style, though. @orta thoughts?

Reply:
It doesn't look like it's being mutated, so I don't think it needs the mental overhead of being put in an each, personally